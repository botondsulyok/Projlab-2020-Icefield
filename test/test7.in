field unstable 2 true 0
stats unstable
entity e1 Eskimo unstable
entity e2 Eskimo unstable
entity e3 Eskimo unstable
entity e4 Eskimo unstable
stats unstable