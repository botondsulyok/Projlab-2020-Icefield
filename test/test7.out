id: unstable
type: unstable
building: plains0
snow: 0
pickable: null
standers: null
pullers: null
masters: null
neighbours: null
capacity: 2
broken: true

Game Over
id: unstable
type: unstable
building: plains0
snow: 0
pickable: null
standers: e1, e2, e3, e4
pullers: e1, e2, e3, e4
masters: e1, e2, e3, e4
neighbours: null
capacity: 2
broken: true

